examples/website-demos/virtual-event/src/components/CentralPanel/rooms.ts

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import { Card1, Card2, Card4, Card5, Card6 } from '../../assets';
+import { Card7 } from '../../assets/eventCardPictures/Card7';
+
+type Room = {
+  chatType: 'main-event' | 'room';
+  content: string;
+  eventName: string;
+  label: string;
+  presenters: number;
+  title: string;
+  Image?: React.FC<{}>;
+  viewers?: number;
+};
+
+export const rooms: Room[] = [
+  {
+    chatType: 'room',
+    content:
+      'ESG Regulations, Standards and Disclosure: Who to publish it for and how to make it meaningful.',
+    eventName: 'esg',
+    Image: Card1,
+    label: 'Private',
+    presenters: 2,
+    title: 'ESG Data - How to create it',
+    viewers: 150,
+  },
+  {
+    chatType: 'room',
+    content:
+      'How to set a business plans to use information to a competitive advantage and support enterprise goals.',
+    eventName: 'qa',
+    Image: Card2,
+    label: 'Moderated',
+    presenters: 6,
+    title: 'Q&A session: Data strategy and executive communication',
+    viewers: 150,
+  },
+
+  {
+    chatType: 'room',
+    content:
+      'How to set a business plans to use information to a competitive advantage and support enterprise goals.',
+    eventName: 'qa',
+    Image: Card5,
+    label: 'Moderated',
+    presenters: 2,
+    title: 'Q&A session: Data strategy and executive communication',
+    viewers: 86,
+  },
+
```

Comment:
event names and content are identical here

Reply:
Yeah the cards are the exact same

Reply:
that's prob a copy/paste error on francesco's side. it won't work well like this since the eventName is the same so it'll trigger the same channel. get updated info from him for one of them